### PR TITLE
Add authorization to Organization Show UI

### PR DESCRIPTION
**Pull request by ncbui**
## What type of PR is this? (check all applicable)

- [] Feature
- [] Refactor
- [] Bug Fix
- [x] Optimization
- [] Documentation Update
- [] Other (describe: )

## Description of what PR does

Shows different view elements and allow access based on whether the user owns the org
- Only users who own the org can see links to 'create a shift', 'edit', and 'delete'.
- Only users who own the org can access the org edit page. 

This is achieved by adding a method `is_owner` to run whenever the controller calls on show edit and update actions. `is_owner` creates an object that can be accessed by the `edit` method in the controller and throughout the app/views/organizations/show.html.erb file.

## Related PRs and/or Issues (if any)
- This is part of steps 3 and 4 in #12 

## QA Instructions, Screenshots, Recordings
- Start the server and navigate to http://127.0.0.1:3000
- View an org while signed in as the organization's owner
- View an same org while signed in as a user who does not own that org

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed _(please include reasons for why tests aren't needed)_
- [ ] no, because I need help
- [ x ] no, they will be added later (please create an Issue for it)

## Added to documentation?

- [ ] Yes, project README
- [ ] No documentation needed

### CODE REVIEW

**Review by Timilehin08 — COMMENTED** [review]
Great job @ncbui! if i login as a worker and not an owner, i don't see create shifts. Also, deleting a shift can't work yet cos we can't successfully create a shift due to the shift start and end not working yet so I guess we can't knw if the delete successfully works yet.

Also, i thought we were styling the create shift with a form similar or same as that of editview ui issue. I guess you're on it. 
Welldone Cam. Reviewing this and having corrections and all on your PRS is also giving me a better grasp of the Workers ui issue.

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -26,6 +27,10 @@ def create
   end
 
   def edit
+    unless is_owner
+        flash[:danger] = "User cannot edit this organization."
```
Comment (msarit) [inline app/controllers/organizations_controller.rb:31]: What about "You are unauthorized to edit this organization"?

I think  it is more descriptive 😄

**Review by msarit — APPROVED** [review]
Looks great! I suggested a small tweak (non-blocking so I will approve anyways)

File: app/controllers/organizations_controller.rb (hunk 1 above)
Comment (ncbui) [reply app/controllers/organizations_controller.rb:31]: @msarit sorry, I'm just seeing this now. Should I still make the tweak to the message?

**Pull request merged**